lake-manifest.json

```diff
@@ -1,7 +1,17 @@
 {"version": "1.1.0",
  "packagesDir": ".lake/packages",
  "packages":
- [{"url": "https://github.com/leanprover-community/plausible",
+ [{"url": "https://github.com/chasenorman/CanonicalLean.git",
```

Comment:
As far as I know, there is no decision about including `canonical` in mathlib, so that you should remove this part.